### PR TITLE
Deprecate `gpt-4-vision-preview` model

### PR DESCRIPTION
**Pull request by tianshanghong**
- Format the file `bot/openai_helper.py`
- Use `gpt-4o` as default vision model

### CODE REVIEW

File: bot/openai_helper.py [hunk 1]
```diff
@@ -26,8 +26,9 @@
 GPT_3_16K_MODELS = ("gpt-3.5-turbo-16k", "gpt-3.5-turbo-16k-0613", "gpt-3.5-turbo-1106", "gpt-3.5-turbo-0125")
 GPT_4_MODELS = ("gpt-4", "gpt-4-0314", "gpt-4-0613", "gpt-4-turbo-preview")
 GPT_4_32K_MODELS = ("gpt-4-32k", "gpt-4-32k-0314", "gpt-4-32k-0613")
-GPT_4_VISION_MODELS = ("gpt-4-vision-preview",)
-GPT_4_128K_MODELS = ("gpt-4-1106-preview","gpt-4-0125-preview","gpt-4-turbo-preview", "gpt-4-turbo", "gpt-4-turbo-2024-04-09")
+GPT_4_VISION_MODELS = ("gpt-4o",)
```
Comment (didyouexpectthat) [inline bot/openai_helper.py:29]: `gpt-4o-mini` is also recommended here

**Comment by didyouexpectthat** [comment]
I implemented these changes and it worked for me. Thank you.

**Comment by p3452** [comment]
Thanks! Now everything is clearly working!

**Comment by moritzfl** [comment]
I can confirm this is working and deployed the bot from [tianshanghong:support-gpt-4o](https://github.com/tianshanghong/chatgpt-telegram-bot/tree/support-gpt-4o) as the vision capability is clearly broken in [n3d1117:main](https://github.com/n3d1117/chatgpt-telegram-bot/tree/main).

It would be great if this PR was merged!

**Comment by n3d1117** [comment]
Thanks @tianshanghong, done in https://github.com/n3d1117/chatgpt-telegram-bot/pull/652

**Pull request closed without merging**